doc/nrf/known_issues.rst

```diff
@@ -483,25 +484,25 @@ KRKNWK-6408: ``diag`` command not supported
 Zigbee
 ======
 
-.. rst-class:: v1-9-1 v1-9-0
+.. rst-class:: v2-0-0 v1-9-1 v1-9-0
 
 KRKNWK-12937: Activation of Sleepy End Device must be done at the very first commissioning procedure for light switch sample
    After programming the :ref:`Zigbee light switch <zigbee_light_switch_sample>` sample and its first commissioning, Zigbee End Device joins the Zigbee network as a normal End Device. Pressing **Button 3** does not switch the device to the Sleepy End Device configuration.
 
    **Workaround:** Keep **Button 3** pressed during the first commissioning procedure.
 
-.. rst-class:: v1-9-1 v1-9-0
+.. rst-class:: v2-0-0 v1-9-1 v1-9-0
 
 KRKNWK-12522: Incorrect Read Attributes Response on reading multiple attributes when the first attribute is unsupported
    When reading multiple attributes at once and the first one is not supported, the Read Attributes Response contains two records for the first supported attribute.
    The first one record has the Status field filled with Unsupported Attribute whereas the second record contains actual data.
 
-.. rst-class:: v1-9-1 v1-9-0
+.. rst-class:: v2-0-0 v1-9-1 v1-9-0
```

Comment:
Valid, but with a new workaround to use ZBOSS 3.12, as per https://projecttools.nordicsemi.no/confluence/display/KRKNWK/Release+notes+and+known+issues+2.0.0 from @maciejpietras -- Maciej, please confirm recommendation to use 3.12 is ok.

Reply:
Issue is still valid for NCS2.0.0 as ZBOSS 3.12 is not there

Reply:
OK, so I'm removing the workaround about using the development version of ZBOSS.